curriculum/js2/8.js

```diff
@@ -4,9 +4,10 @@
  */
 
 const solution = () => {
-  Array.prototype.gsMap = function() {
-    // To get the actual array, use this
-    return [];
+  Array.prototype.gsMap = function(cb, result = []) {
```

Comment:
which algorithm problem is this for?  also, we don't allow built in methods to be used.

Reply:
curriculum/js2/8.js

Reply:
Replicate Array.prototype.map function and call it gsMap